packages/auto-dag-data/README.md

```diff
@@ -36,33 +36,40 @@ yarn add @autonomys/auto-dag-data
 
 ### Creating an IPLD DAG from a File
 
-To create an IPLD DAG from a file, you can use the `createFileIPLDDag` function:
+To create an IPLD DAG from a file, you can use the `processFileToIPLDFormat` function:
 
 ```typescript
-import { createFileIPLDDag } from '@autonomys/auto-dag-data'
+import { processFileToIPLDFormat } from '@autonomys/auto-dag-data'
+import { MemoryBlockstore } from 'blockstore-core/memory'
```

Comment:
if we use blockstore-core inside this package we should expose all functions needed to not need extra dependencies to use this package

Reply:
It's just a devDependency so I'd prefer not to add it to `dependencies`